src/DicomWebClient/API/IQidoService.cs

```diff
@@ -0,0 +1,76 @@
+        /// </summary>
+        IAsyncEnumerable<string> SearchForStudies();
+
+        /// <summary>
+        /// Search for studies based on provided query parameters.
+        /// </summary>
+        /// <param name="queryParameters">A dictionary object where the <c>Key</c> contains the DICOM tag
+        /// or keyword of an attribute and the <c>Value</c> contains the expected value to match.</param>
+        IAsyncEnumerable<string> SearchForStudies(Dictionary<string, string> queryParameters);
+
+        /// <summary>
+        /// Search for studies based on provided query parameters with additional DICOM fields to be included in the response message.
+        /// </summary>
+        /// <param name="queryParameters">A dictionary object where the <c>Key</c> contains the DICOM tag
+        /// or keyword of an attribute and the <c>Value</c> contains the expected value to match.</param>
+        /// <param name="fieldsToInclude">Liist of DICOM tags of name of the DICOM tag to be included in the response.</param>
+        IAsyncEnumerable<string> SearchForStudies(Dictionary<string, string> queryParameters, List<string> fieldsToInclude);
+
+        /// <summary>
+        /// Search for studies based on provided query parameters with additional DICOM fields to be included in the response message.
+        /// </summary>
+        /// <param name="queryParameters">A dictionary object where the <c>Key</c> contains the DICOM tag
+        /// or keyword of an attribute and the <c>Value</c> contains the expected value to match.</param>
+        /// <param name="fieldsToInclude">Liist of DICOM tags of name of the DICOM tag to be included in the response.</param>
+        /// <param name="fuzzyMatching">Whether fuzzy semantic matching should be performed.</param>
+        IAsyncEnumerable<string> SearchForStudies(Dictionary<string, string> queryParameters, List<string> fieldsToInclude, bool fuzzyMatching);
+
+        /// <summary>
+        /// Search for studies based on provided query parameters with additional DICOM fields to be included in the response message.
+        /// </summary>
+        /// <param name="queryParameters">A dictionary object where the <c>Key</c> contains the DICOM tag
+        /// or keyword of an attribute and the <c>Value</c> contains the expected value to match.</param>
+        /// <param name="fieldsToInclude">Liist of DICOM tags of name of the DICOM tag to be included in the response.</param>
+        /// <param name="limit">Maximum number of results to be returned.</param>
+        IAsyncEnumerable<string> SearchForStudies(Dictionary<string, string> queryParameters, List<string> fieldsToInclude, bool fuzzyMatching, int limit);
+
+        /// <summary>
+        /// Search for studies based on provided query parameters with additional DICOM fields to be included in the response message.
+        /// </summary>
+        /// <param name="queryParameters">A dictionary object where the <c>Key</c> contains the DICOM tag
```

Comment:
as far as I know, the `<c>` tag isn't recognized by Intellisense nor Roslyn, meaning it serves little to no value.

tags like `<paramref name="Value"/>` will both be recognized by Intellisense for easy developer consumption, and by Roslyn for refactoring.

Reply:
`<c>` does get highlighted by docfx.